.eslintrc

```diff
@@ -20,9 +26,20 @@
       "version": "16.5"
     }
   },
-  "plugins": ["react"],
   "rules": {
     "no-console": 2,
-    "react/prop-types": 1
+    "react/prop-types": 1,
+    "semi": [
+      2,
+      "always"
+    ],
+    "no-extra-semi": 2,
+    "strict": [
+      "error",
+      "global"
+    ],
+    "no-unused-vars": "error",
+    "prefer-const": "warn",
```

Comment:
I added this one and the next one because it required too much changes in the code, you may want to review your `let` usage and the dot-notation (things like `avatar['sm']` can't change to `avatar.sm`)